benchmark/inclusion/run.sh

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
 
 # global refinement:
-for r in "3" "4" "5" "6" "7" "8"
+echo "--Global Refinement--"
+for r in "4" "5" "6" "7" "8" "9"
 do
 echo "ref $r:"
 cp global.prm.base temp.prm
-echo "subsection Mesh refinement" >>temp.prm
+echo "set Output directory = output/global/ref$r" >> temp.prm
+echo "subsection Mesh refinement" >> temp.prm
 echo "set Initial global refinement = $r" >> temp.prm
 echo "end" >> temp.prm
-./aspect temp.prm | grep Error
+./aspect temp.prm | grep DoFs
```

Comment:
I think having everything in one line is very convenient (copy&paste into a spreadsheet etc).